src/routes/politicians/[id]/+page.svelte

```diff
@@ -91,6 +105,12 @@
 								<a class="text-black" href="/">{post.organizations[0].name}</a>
 							</li>
 						{/each}
+						{#if roleAliases[currentPartyRole] && currentParty?.name}
```

Comment:
If there is no role, that might be the case that we know they are a party member, but we just didn't know the role.

So if there is no role, we just don't show the role, but we should still show the party name.